source/uwp/AdaptiveCardsObjectModel/lib/AdaptiveColumn.cpp

```diff
@@ -30,7 +31,10 @@ namespace winrt::AdaptiveCards::ObjectModel::Uwp::implementation
             BackgroundImage = winrt::make<implementation::AdaptiveBackgroundImage>(backgroundImage);
         }
 
-        Rtl = sharedColumn->GetRtl();
+        if (sharedColumn->GetRtl().has_value())
+        {
+            Rtl = winrt::box_value(sharedColumn->GetRtl().value()).as<Windows::Foundation::IReference<bool>>();
```

Comment:
Note: coming soon, the `wrl::winrt::property<std::optional<bool>>` will do this conversion for you so you can just say `Rtl = sharedColumn->GetRtl()` in one line.

Reply:
This goes away with the property_opt fix.